caf-kubernetes-client/src/main/java/com/github/cafapi/kubernetes/client/KubernetesClientFactory.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2024 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafapi.kubernetes.client;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.jackson.internal.jackson.jaxrs.json.JacksonJsonProvider;
+import org.glassfish.jersey.jdk.connector.JdkConnectorProvider;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.github.cafapi.kubernetes.client.client.ApiClient;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+
+/**
+ * This class is loosely based on
+ * <a href="https://github.com/kubernetes-client/java/blob/automated-release-21.0.1/util/src/main/java/io/kubernetes/client/util/ClientBuilder.java#L70">io.kubernetes.client.util.ClientBuilder.java</a>,
+ * which we stopped using because it brought in a lot of unwanted dependencies (such as the AWS SDK, Google Protocol Buffers, and the
+ * Bouncy Castle libraries).
```

Comment:
Drop the line number and the rationale.

```suggestion
 * This class is loosely based on
 * <a href="https://github.com/kubernetes-client/java/blob/v21.0.1/util/src/main/java/io/kubernetes/client/util/ClientBuilder.java">io.kubernetes.client.util.ClientBuilder.java</a>.
```

Reply:
From the license terms of the code you've referenced I believe it is necessary to include their copyright notice in this file also, which I think is just "Copyright 2020 The Kubernetes Authors."

[`https://github.com/kubernetes-client/java/blob/v21.0.1/LICENSE#L100-L102`](https://github.com/kubernetes-client/java/blob/v21.0.1/LICENSE#L100-L102)